applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,104 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Desktop Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
+DVFS is critical for balancing performance and power consumption, especially in battery-operated devices.
+
+In the nRF Desktop, the DVFS module enables the system to use the lowest possible frequency and voltage when the application performs less computation-challenging activities such as Bluetooth LE advertising.
```

Comment:
Can we rephrase it a bit to something like:
`By default, system uses the lowest available frequency to save power. If application performs computation-consuming tasks, the DVFS module will increase frequency for better performance.`?

That would be more in line with the implementation

Reply:
It is not true what you are writing: "By default we use 320MHz so that's why I would state that we lower frequency in some cases and not the other way around."

Reply:
I would focus on the use-case where the module is enabled and actively controls the frequency here.
Maybe we could rephrase a bit to prevent misunderstandings: `By default, the module sets the lowest available frequency to save power. ...`

Reply:
You can still mention that disabling the module leads to using `320MHz` frequency all the time (IMHO this is out of scope of this doc file)

Reply:
> I would focus on the use-case where the module is enabled and actively controls the frequency here. Maybe we could rephrase a bit to prevent misunderstandings: `By default, the module sets the lowest available frequency to save power. ...`

It is thea same as this sentence: "In the nRF Desktop, the DVFS module enables the system to use the lowest possible frequency and voltage when the application performs less computation-challenging activities such as Bluetooth LE advertising."

It is just more generic

Reply:
I would not insist here, we can keep it as is. I just tried to make this part more in line with the implementation.